Gems/Atom/RHI/Code/Include/Atom/RHI/DispatchRaysItem.h

```diff
@@ -21,15 +21,72 @@ namespace AZ
         class ImageView;
         class BufferView;
 
+        //! Arguments used when submitting a (direct) dispatch rays call into a CommandList.
+        struct DispatchRaysDirect
+        {
+            DispatchRaysDirect() = default;
+
+            DispatchRaysDirect(uint32_t width, uint32_t height, uint32_t depth)
+                : m_width(width)
+                , m_height(height)
+                , m_depth(depth)
+            {
+            }
+
+            uint32_t m_width = 1;
+            uint32_t m_height = 1;
+            uint32_t m_depth = 1;
+        };
+
+        //! Arguments used when submitting an indirect dispatch rays call into a CommandList.
+        //! The indirect dispatch arguments are the same ones as the indirect draw ones.
+        using DispatchRaysIndirect = IndirectArguments;
+
+        enum class DispatchRaysType : uint8_t
+        {
+            Direct = 0, // A dispatch rays call where the arguments are pass directly to the submit function.
+            Indirect // An indirect dispatch rays call that uses a buffer that contains the arguments.
+        };
+
+        //! Encapsulates the arguments that are specific to a type of dispatch.
+        //! It uses a union to be able to store all possible arguments.
+        struct DispatchRaysArguments
+        {
+            AZ_TYPE_INFO(DispatchArguments, "F8BE4C19-F35D-4545-B17F-3C2B4D7EF4FF");
```

Comment:
It should be `DispatchRaysArguments` in `AZ_TYPE_INFO(`

Reply:
Correct. Please go ahead and make the change. Thanks.